docs/_docs/languages/hack.md

```diff
@@ -147,73 +141,82 @@ xdebug.remote_port = 9000
 ```
 
 > In the Nuclide Settings, there is an option to specify the remote port as well. If you specify
-> the port in an ini file via `xdebug.remote_port`, make sure that setting matches what is in the
-> Nuclide setting.
+> the port in an `.ini` file with `xdebug.remote_port`, make sure it matches what is in the
+> Nuclide port setting found under `nuclide-debugger-php`.
+
+Make sure the **Arguments for your PHP runtime** setting points to your `.ini` file.
+
+***Example***
+
+```bash
+-c /root/docker-shared/code/xdebug.ini
+```
+
+The **Path to your PHP runtime** setting should indicate the location of HHVM on your server, such as `/usr/bin/hhvm`.
+
 
 ### Debugging: HHVM Toolbar
 
-Nuclide provides an HHVM toolbar. You can launch the toolbar from the
-[Nuclide toolbar](/docs/features/toolbar/#buttons) or from the
-[command-palette](/docs/editor/basics/#command-palette) with `Nuclide HHVM Toolbar: Toggle`.
+<!-- HHVM TOOLBAR WILL BE MOVING INTO TASK RUNNER -->
```

Comment:
I assume you meant to keep this here for future reference? I think it is ok to have it as a comment.